src/styles/3_components/_buttons.css

```diff
@@ -0,0 +1,73 @@
+@layer components {
+    /* Primary */
+    .tw-btn-primary {
+        @apply tw-relative tw-leading-none tw-inline-block tw-whitespace-nowrap tw-select-none tw-py-3 tw-px-4 tw-rounded-full tw-border-2 tw-border-transparent tw-font-roboto-condensed tw-font-bold tw-uppercase tw-cursor-pointer tw-transition tw-text-white hover:tw-bg-gray-100 active:tw-bg-current active:tw-text-white visited:tw-text-white hover:tw-no-underline;
+    }
+    .tw-btn-primary.tw-bg-drumeo:hover{
+        @apply tw-bg-drumeo-light !important;
+    }
+    .tw-btn-primary.tw-bg-guitareo:hover{
+        @apply tw-bg-guitareo-light !important;
+    }
+    .tw-btn-primary.tw-bg-pianote:hover{
+        @apply tw-bg-pianote-light !important;
+    }
+    .tw-btn-primary[disabled] {
+        @apply tw-pointer-events-none tw-bg-gray-200 tw-text-gray-400 !important;
+    }
+
+    /* Secondary */
+    .tw-btn-secondary {
+        @apply tw-relative tw-leading-none tw-inline-block tw-whitespace-nowrap tw-select-none tw-py-3 tw-px-4 tw-rounded-full tw-border-2 tw-border-current tw-font-roboto-condensed tw-font-bold tw-uppercase tw-cursor-pointer tw-transition hover:tw-bg-gray-100 active:tw-bg-current hover:tw-no-underline;
+    }
+    .tw-btn-secondary:active span,
+    .tw-btn-secondary:active i {
```

Comment:
i cant quite tell from the view here, but does this include hover effects for outline buttons? i know Jord really wanted to add that for our members areas!

Reply:
Yes sir! take a look at the end, `hover:tw-bg-gray-100 active:tw-bg-current hover:tw-no-underline`. But also, these styles can be overridden in the template because these utilities aren't `!important` so your new classes will take priority if you need to add a different hover state.